### PR TITLE
DBZ-4109 DBZ-4234 Upgrade Debezium and other dependencies

### PR DESCRIPTION
**Pull request by rk3rn3r**
DBZ-4109 Upgrade Debezium and other dependencies (Quarkus, etc) of Debezium UI

closes https://issues.redhat.com/browse/DBZ-4109

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@rk3rn3r, where are we with this one? What's missing in order to wrap it up?

File: backend/pom.xml [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <name>Debezium UI Backend</name>
 
   <properties>
+    <quarkus.package.type>uber-jar</quarkus.package.type>
```
Comment (gunnarmorling) [inline backend/pom.xml:17]: Oh, what's the reason for the uber-JAR? This was consciously using the multi-layer approach before.

**Review by gunnarmorling — COMMENTED** [review]
The dep updates LGTM, but why are we moving to an uber -JAR? It's generally not a recommended practice for container-based deployments.

File: backend/pom.xml (hunk 1 above)
Comment (rk3rn3r) [reply backend/pom.xml:17]: See https://issues.redhat.com/browse/DBZ-4621

**Pull request closed without merging**